components/l3mbda/sources/common/webhook.mjs

```diff
@@ -0,0 +1,126 @@
+import app from "../../l3mbda.app.mjs";
+import { ConfigurationError } from "@pipedream/platform";
+
+export default {
+  props: {
+    app,
+    db: "$.service.db",
+    http: {
+      type: "$.interface.http",
+      customResponse: true,
+    },
+    from: {
+      propDefinition: [
+        app,
+        "from",
+      ],
+    },
+    to: {
+      propDefinition: [
+        app,
+        "to",
+      ],
+    },
+    token: {
+      propDefinition: [
+        app,
+        "token",
+      ],
+    },
```

Comment:
The prop `token` is declared in `webhook.mjs`, and also in `new-erc-1115-transfer-instant`, `new-erc-20-transfer-instant`, and `new-erc-721-transfer-instant`. It looks like maybe it can be removed from `webhook.mjs`?
```suggestion
```